lib/tool_belt/commands/procedure.rb

```diff
@@ -36,6 +36,7 @@ def execute
             debian_full_version: debian_full_version, # 1.20.0~rc1-1
             full_version: full_version, # 1.20.0-RC1
             version: version, # 1.20.0
+            short_version: "#{major}.#{minor}",
```

Comment:
We already have this as `branch`. Does the duplication make sense? Is branch badly named?

Reply:
Hrmm, fair, though I would have expected branch to be `1.20-stable` since that is the branch name in Foreman. Should I update it to that? or set branch based on project?

Reply:
Naming is hard. I guess that makes sense. I'd be fine with renaming `branch` to `short_version`. Since currently there's no use for `branch` (`1.20-stable`) I'd prefer to omit it until we need it.